spec/std/time/time_spec.cr

```diff
@@ -597,6 +597,20 @@ describe Time do
     (time.to_utc <=> time).should eq(0)
   end
 
+  it %(changes timezone with ENV["TZ"]) do
+    begin
+      ENV["TZ"] = "America/New_York"
+      offset1 = Time.local_offset_in_minutes
+
+      ENV["TZ"] = "Europe/Berlin"
+      offset2 = Time.local_offset_in_minutes
+
+      offset1.should_not eq(offset2)
+    ensure
+      ENV["TZ"] = ""
```

Comment:
Shouldn't `TZ` be restored to the previous value instead of set to empty string?